WORKSPACE

```diff
@@ -46,10 +46,10 @@ android_workspace()
 # will trigger bazel failure when Android SDK is not configured.
 # The targets in the following folder need to be included in APK and will
 # be invoked by binder transport implementation through JNI.
-local_repository(
-    name = "binder_transport_android_helper",
-    path = "./src/core/ext/transport/binder/java",
-)
+#local_repository(
```

Comment:
nit: Commented out code is discouraged.

Reply:
oh forgot this TODO comment. This needs to be addressed by follow-up.